packages/api/cms-api/src/dam/files/files.service.ts

```diff
@@ -240,6 +240,95 @@ export class FilesService {
+            try {
+                image = await probe(createReadStream(file.path));
+                if (image.type == "svg") image = undefined;
+                if (image !== undefined && image.orientation !== undefined && [6, 8].includes(image.orientation)) {
+                    image = {
+                        ...image,
+                        width: image.height,
+                        height: image.width,
+                    };
+                }
+            } catch (e) {
+                // empty
+            }
+
+            if (
+                image !== undefined &&
+                image.width &&
+                image.height &&
+                Math.round(((image.width * image.height) / 1000000) * 10) / 10 >= this.imgproxyConfig.maxSrcResolution
+            ) {
+                throw new CometImageResolutionException(`Maximal image resolution exceeded`);
+            }
+
+            if (folderId) {
+                const folder = await this.foldersService.findOneById(folderId);
+                if (!folder) throw new Error("Folder not found");
+                if (!this.contentScopeService.scopesAreEqual(folder.scope, scope)) {
+                    throw new Error("Folder scope doesn't match passed scope");
+                }
+            }
+
+            let exifData: Record<string, string | number | Uint8Array | number[] | Uint16Array> | undefined;
+            if (exifrSupportedMimetypes.includes(file.mimetype)) {
+                exifData = await exifr.parse(file.path);
+            }
+
+            const exisitingFile = await this.findOneByFilenameAndFolder({ filename: file.originalname, folderId });
+            if (!exisitingFile) throw new Error("File not found");
+
+            await this.blobStorageBackendService.upload(file, contentHash, this.config.filesDirectory);
+
+            // Check if the current file is the only one using the contentHash before deleting from blob storage
+            if (
+                (await withFilesSelect(this.filesRepository.createQueryBuilder("file"), { contentHash: exisitingFile.contentHash }).getResult())
+                    .length === 1
+            ) {
+                await this.blobStorageBackendService.removeFile(this.config.filesDirectory, createHashedPath(exisitingFile.contentHash));
+            }
+
+            if (
+                image &&
+                image.width &&
+                image.height &&
+                exisitingFile.image &&
+                (image.height !== exisitingFile.image.height || image.width !== exisitingFile.image.width)
+            ) {
+                exisitingFile.image.width = image.width;
+                exisitingFile.image.height = image.height;
+                exisitingFile.image.exif = exifData;
+
+                if (exisitingFile.image.cropArea.focalPoint !== FocalPoint.SMART) {
+                    exisitingFile.image.cropArea = { focalPoint: FocalPoint.SMART };
+                }
```

Comment:
Did we explicitly decide to reset the crop area in the DAM? Seems like a good idea to me, but could be unexpected for users.

Reply:
wasn't that how we discussed it? 🤔  @thomasdax98 also wrote it into the ticket

Reply:
Actually, I said that we should remove it if the aspect ratio is different. This removes the cropArea always.

But I changed my mind about that. Since we explicitly state in the dialog that we don't touch the cropArea (because we can't in the blocks), I think we also shouldn't touch it in the DAM. Everything else is hard to explain. 

I changed this in 59fa409647c1a708b5c1b1257933b43fd0353ba2